### ISSUE TITLE
Benefits from this mixed approach?

### ISSUE THREAD

**Issue by kristianmandrup** [RichiCoder1/Aurelia-React-PoC#1]
I'm curious if you find this mixed approach beneficial? When would you consider using Aurelia with React? I guess some for components it makes sense to leverage React rendering? 
React has a very different philosophy with Flow and Flux. Could these be mixed with Aurelia as well for more benefits? Your thoughts ;)

**Comment by RichiCoder1** [comment]
Having done this approach, I find React redundant, if not inconvenient, to use with Aurelia. Aurelia's custom elements are about as powerful and give you many of the same benefits that React has, and about the only points I'd give in React's favor is it's better life cycle and JSX, and the latter is arguable. Aurelia's databinding, structure, and separation is just straight up nicer, and much databinding specifically is much easier to use if you're just using Aurelia elements rather than trying to shoehorn in the data through react's props.

tl;dr: React adds little, if no value.
